### PR TITLE
[MM-58408] Migrate tooltips of 'components/file_preview_modal/file_preview_modal_main_nav/file_preview_modal_main_nav' to WithTooltip

### PR DESCRIPTION
**Pull request by Camillarhi**
#### Summary

Migrate tooltips to use WithTooltip in 'components/file_preview_modal/file_preview_modal_main_nav/file_preview_modal_main_nav' file

#### Ticket Link

Fixes https://github.com/mattermost/mattermost/issues/27155
Jira https://mattermost.atlassian.net/browse/MM-58408

#### Screenshots
| image | image|

| image | image|

#### Release Note

```
NONE
```

### CODE REVIEW

File: webapp/channels/src/components/file_preview_modal/file_preview_modal_main_nav/file_preview_modal_main_nav.tsx [hunk 1]
```diff
@@ -20,51 +17,41 @@ interface Props {
 
 const FilePreviewModalMainNav: React.FC<Props> = (props: Props) => {
     const leftArrow = (
-        <OverlayTrigger
-            delayShow={Constants.OVERLAY_TIME_DELAY}
-            key='previewArrowLeft'
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/file_preview_modal/file_preview_modal_main_nav/file_preview_modal_main_nav.tsx:25]: @Camillarhi lets have the key prop back as it was before

File: webapp/channels/src/components/file_preview_modal/file_preview_modal_main_nav/file_preview_modal_main_nav.tsx [hunk 2]
```diff
@@ -20,51 +17,41 @@ interface Props {
...
+                >
+                    <i className='icon icon-chevron-left'/>
+                </button>                         
+        </WithTooltip>
     );
 
     const rightArrow = (
-        <OverlayTrigger
-            delayShow={Constants.OVERLAY_TIME_DELAY}
-            key='publicLink'
```
Comment (M-ZubairAhmed) [inline webapp/channels/src/components/file_preview_modal/file_preview_modal_main_nav/file_preview_modal_main_nav.tsx:49]: same key from before here as well

**Review by M-ZubairAhmed — CHANGES_REQUESTED** [review]
Hey @Camillarhi to update the test snapshots you can
`npm run test:updatesnapshot` and to automatically fix the eslint errors you can `npm run fix`

**Review by Rajat-Dabade — APPROVED** [review]

**Review by M-ZubairAhmed — APPROVED** [review]
This looks good :shipit: Thank you

**Review by yasserfaraazkhan — APPROVED** [review]

**Pull request merged**